webform.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/fullstorydev/grpcui/internal/resources/webform"
 )
 
-var webFormTemplate = template.Must(template.New("grpc web form").Parse(string(webform.WebFormTemplate())))
+var webFormTemplate = template.Must(template.New("grpc web form").Parse(string(webform.MustAsset("webform-template.html"))))
```

Comment:
Hmm, that's gross. So the new gobindata doesn't generate functions you can use? It's a shame to lose the type/compile-time safety and have to revert to runtime string checks...

Maybe add provide such functions, hand-written, and add a test that all of the functions return data w/out panic'ing?